x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/value_lists/value_list_items.cy.ts

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { login } from '../../../../tasks/login';
+import { visit } from '../../../../tasks/navigation';
+import { TOASTER_BODY } from '../../../../screens/alerts_detection_rules';
+import { closeErrorToast } from '../../../../tasks/alerts_detection_rules';
+import {
+  createListsIndex,
+  waitForValueListsModalToBeLoaded,
+  openValueListsModal,
+  importValueList,
+  waitForListsIndex,
+  deleteValueLists,
+  KNOWN_VALUE_LIST_FILES,
+  openValueListItemsModal,
+  searchValueListItemsModal,
+  getValueListItemsTableRow,
+  checkTotalItems,
+  deleteListItem,
+  clearSearchValueListItemsModal,
+  sortByValue,
+  updateListItem,
+  addListItem,
+  selectValueListsItemsFile,
+  uploadValueListsItemsFile,
+  mockFetchListItemsError,
+  mockCreateListItemError,
+  mockUpdateListItemError,
+  mockDeleteListItemError,
+} from '../../../../tasks/lists';
+import {
+  VALUE_LIST_ITEMS_MODAL_INFO,
+  VALUE_LIST_ITEMS_MODAL_TABLE,
+  VALUE_LIST_ITEMS_MODAL_TITLE,
+} from '../../../../screens/lists';
+import { RULES_MANAGEMENT_URL } from '../../../../urls/rules_management';
+
+describe(
+  'Value list items',
+  {
+    tags: ['@ess', '@serverless'],
+    env: {
+      ftrConfig: {
+        kbnServerArgs: [
+          `--xpack.securitySolution.enableExperimental=${JSON.stringify(['valueListItemsModal'])}`,
+        ],
+      },
+    },
+  },
+  () => {
+    beforeEach(() => {
+      login();
+      deleteValueLists([
+        KNOWN_VALUE_LIST_FILES.TEXT,
+        KNOWN_VALUE_LIST_FILES.IPs,
+        KNOWN_VALUE_LIST_FILES.CIDRs,
+      ]);
+      createListsIndex();
+      visit(RULES_MANAGEMENT_URL);
+      waitForListsIndex();
+      waitForValueListsModalToBeLoaded();
+      importValueList(KNOWN_VALUE_LIST_FILES.TEXT, 'keyword');
+      openValueListsModal();
+    });
+
+    afterEach(() => {
+      deleteValueLists([
+        KNOWN_VALUE_LIST_FILES.TEXT,
+        KNOWN_VALUE_LIST_FILES.IPs,
+        KNOWN_VALUE_LIST_FILES.CIDRs,
+      ]);
+    });
+
+    it('can CRUD value list items', () => {
+      openValueListItemsModal(KNOWN_VALUE_LIST_FILES.TEXT);
+      const totalItems = 12;
+      const perPage = 10;
+
+      // check modal title and info
+      cy.get(VALUE_LIST_ITEMS_MODAL_TITLE).should('have.text', KNOWN_VALUE_LIST_FILES.TEXT);
+      cy.get(VALUE_LIST_ITEMS_MODAL_INFO).contains('Type: keyword');
+      cy.get(VALUE_LIST_ITEMS_MODAL_INFO).contains('Updated by: system_indices_superuse');
+      checkTotalItems(totalItems);
```

Comment:
NIT: It is preferred to not wrap assertions.

Reply:
If I use this assertions 5 times, what's a better approach?
`cy.get(VALUE_LIST_ITEMS_MODAL_INFO).contains(`Total items: ${totalItems}`)`

Reply:
@MadameSheema it would be helpful to provide reasoning for these suggestions (even if it's just linking to an article, etc.)

Reply:
@rylnd @nkhristinin it is a preference from several developers, it is a `NIT` not a blocker. The preference about not having it wrapped is for in case of failure to see quickly what is happening, but at the end, is up to your team to decide.